versioned_docs/version-v0.14.0/int/key-concepts.md

```diff
@@ -89,6 +89,20 @@ A distributed validator key is a group of BLS private keys, that together operat
 
 One piece of the distributed validator private key.
 
+### Distributed Validator Threshold
+
+The number of nodes in a cluster that needs to be online and honest for their distributed validators to be online is outlined in the following table.
+
+| Cluster Size | Threshold | Note              |
+|:------------:|:---------:|:------------------|
+| 4            |  3/4      | Minimum threshold |
+| 5            |  4/5      |                   |
+| 6            |  4/6      | Minimum to tolerate two offline nodes|
+| 7            |  5/7      | Minimum to tolerate two **malicious** nodes |
+| 8            |  6/8      |                   |
+| 9            |  6/9      | Minimum to tolerate three offline nodes |
+| 10           |  7/10     | Minimum to tolerate three **malicious** nodes |
```

Comment:
should threshold column be like 3-4, 4-5, 4-6, etc? Range of acceptable values.

Reply:
Not sure, it was already added in https://github.com/ObolNetwork/obol-docs/pull/142/files